### ISSUE TITLE
FoV is completely borked in 1440p and 2160p

### ISSUE THREAD

**Issue by pagb666** [LTCHIPS/rottexpr#60]
Well, it's not only that the FoV seems wrong, but like if the locales were many times bigger than they actually are. I've included a couple videos for 1440p and 4K as examples. This is with the FoV slider set to the minimum.
4K: https://youtu.be/pm5-cbFmF5E
1440p: https://youtu.be/F0CjxoJP0w0

**Comment by eggmanstudio** [comment]
Same issue. Also, in my case the game crashes after about 5 seconds.

**Comment by Calinou** [comment]
This can be worked around by increasing the FOV slider until things look correct.

It's located in **Options > Visual Options > Adjust Focal Width**.

Example for 2560×1440:

![2022-04-30_01 48 39](https://user-images.githubusercontent.com/180032/166081506-0451cbc2-25a4-4fd6-8e52-ca2e5611b4fd.png)

Here's how it looks in practice:

![2022-04-30_01 49 06](https://user-images.githubusercontent.com/180032/166081508-ce69b711-0bc4-425d-9988-fa9a79f60c2b.png)

**Comment by pagb666** [comment]
Gonna try right now. Thanks!

**Comment by pagb666** [comment]
Yes, it works for 1440p. But I can't find a proper slider spot for 4K, I can get an aproximation where FoV is a bit high, but there are some ticks of the bar where the world rendering is all garbled and game crashes after a few seconds.
